src/Groups/action.jl

```diff
@@ -75,11 +75,11 @@ julia> (1, 2, 4)^g[1]
 """
 on_tuples(tuple::GapObj, x::GAPGroupElem) = GAPWrap.OnTuples(tuple, GapObj(x))
 
-on_tuples(tuple::Vector{T}, x::GAPGroupElem) where T = T[pnt^x for pnt in tuple]
-^(tuple::Vector{T}, x::GAPGroupElem) where T = on_tuples(tuple, x)
+on_tuples(tuple::Vector{T}, x::S) where {T, S <: GroupElem} = T[pnt^x for pnt in tuple]
+^(tuple::Vector{T}, x::S) where {T, S <: GroupElem} = on_tuples(tuple, x)
```

Comment:
As we just discussed, maybe simplify this to:
```suggestion
on_tuples(tuple::Vector{T}, x::GroupElem) where T = T[pnt^x for pnt in tuple]
^(tuple::Vector{T}, x::GroupElem) where T = on_tuples(tuple, x)
```

But let's first see what the CI tests report